inst/htmljs/animint.js

```diff
@@ -1842,11 +1842,11 @@ var animint = function (to_select, json_file) {
 	  return d["clickSelects.variable"] + " " + d["clickSelects.value"];
 	};
       }
-      var tooltip = d3.select("#plot").select(".animint-tooltip").node() 
-    ? d3.select(".animint-tooltip")
-    : d3.select("#plot").append("div")
-        .attr("class", "animint-tooltip")
-        .style("opacity", 0);
+      var tooltip = d3.select(to_select).select(".animint-tooltip").node()
```

Comment:
`to_select` may be something other than `#plot` (depending on the R code chunk name) in an Rmd doc.